src/CsrValidation/csharp/ScepValidation/IntuneScepServiceException.cs

```diff
@@ -119,5 +120,19 @@ public enum ErrorCode
                 throw;
             }
         }
+
+        public override void GetObjectData(SerializationInfo info, StreamingContext context)
+        {
+            if (info == null)
+            {
```

Comment:
I don't think this is necessary.  I don't see any examples of it and it would derail the original exception.